src/geometry_header.F90

```diff
@@ -7,13 +7,15 @@ module geometry_header
 !===============================================================================
 
   type Universe
-     integer :: id                    ! Unique ID
-     integer :: type                  ! Type
-     integer :: n_cells               ! # of cells within
-     integer, allocatable :: cells(:) ! List of cells within
-     real(8) :: x0                    ! Translation in x-coordinate
-     real(8) :: y0                    ! Translation in y-coordinate
-     real(8) :: z0                    ! Translation in z-coordinate
+     integer :: id                     ! Unique ID
+     integer :: type                   ! Type
+     integer :: n_cells                ! # of cells within
+     integer, allocatable :: cells(:)  ! List of cells within
+     real(8) :: x0                     ! Translation in x-coordinate
+     real(8) :: y0                     ! Translation in y-coordinate
+     real(8) :: z0                     ! Translation in z-coordinate
+     integer, allocatable :: kount(:)  ! Distribcell target counts
+     logical, allocatable :: search(:) ! Distribcell search completed
```

Comment:
Can we make the `kount` and `search` arrays local variables in the `prepare_distribcell` subroutine?  I don't like carrying them around throughout the entire simulation when we only need them during initialization (even though they are deallocated).  I think we could pass the `kount` and `search` arrays as arguments to `calc_offsets`.